docs/user-guide/how_to_schedule_multipe_single_pods_with_podgroup.md

```diff
@@ -0,0 +1,99 @@
+# Manage & Schedule multiple single pods with podgroup
+
+## Background
+Volcano can manage single pod when the scheduler of pod is volcano, volcano controller will help to create a podgroup for this pod.
+But it can not manage and schedule multiple single pods in one pg now. And this missing ability is very useful in spark client mode and so on.
+It will also play a role in some scenarios that require multiple single pods to work together (vj cannot be used).
+
+## Key Points
```

Comment:
In the current PR implementation, users are allowed to add MinResource resource information through annotation, but podgroup has many attributes, such as: MinMember, Queue, etc. The PR does not support the setting of these information. Can you add in-range and out-of-range in the document? Instructions are easier for users to understand.

Reply:
I would add some limitation to the current